pkg/plugins/policies/meshloadbalancingstrategy/plugin/v1alpha1/priority.go

```diff
@@ -69,15 +69,15 @@ func getCrossZoneLbGroups(conf *api.Conf, localZone string) []CrossZoneLbGroup {
 			case api.AnyExcept:
 				lb.Type = api.AnyExcept
 				zones := map[string]bool{}
-				for _, zone := range rule.To.Zones {
+				for _, zone := range *rule.To.Zones {
 					zones[zone] = true
 				}
 				lb.Zones = zones
 				lb.Priority = uint32(priority + 1)
 			case api.Only:
 				lb.Type = api.Only
 				zones := map[string]bool{}
-				for _, zone := range rule.To.Zones {
+				for _, zone := range *rule.To.Zones {
```

Comment:
Don't we need to check if isn't nul `*rule.To.Zones`

Reply:
validation on API is checking this, can there be a situation when this will be nil? After some merging?

Reply:
Not sure if that can happen but would be good to avoid nil pointer if it's somehow possible

Reply:
added check